addons/base_import/static/src/xml/import.xml

```diff
@@ -6,21 +6,30 @@
             <input type="hidden" name="session_id"
                t-att-value="widget.session.session_id"/>
             <input type="hidden" name="import_id"/>
-            <p>Select the file to import. If you need a sample importable file, you
-            can use the export tool to generate one.</p>
-            <div class="oe_import_box">
-                <label t-attf-for="file_#{_id}" autofocus="autofocus">File:</label>
-                <input type="file" id-attf-id="file_#{_id}"
-                       name="file" class="oe_import_file" accept=".csv, .xls, .xlsx, .ods"/>
-                <button type="button" class="oe_import_file_reload"
-                        disabled="disabled"
-                        title="Reload data to check changes.">
-                    <img src="/web/static/src/img/icons/gtk-refresh.png"/>
-                </button>
-                <div class="oe_import_with_file">
+            <div class="oe_import_box col-sm-9">
+                <div class="col-sm-12">
+                    <p>Select a CSV or Excel file to import. <a href="https://github.com/odoo/documentation-user" target="new" class="pull-right">Help</a></p>
```

Comment:
That's a completely worthless link, the user documentation does not help anything.